src/App.js

```diff
@@ -1,27 +1,76 @@
-import React from 'react'
+import React, { useState } from 'react'
 import Nav from './Nav'
 import Homepage from './Homepage'
 import Login from './Login'
 import About from './About'
 
 import { Route, Switch } from 'react-router-dom';
+import { gql, useQuery, useMutation } from '@apollo/client';
+
 const App = () => {
 
+const [user, setUser] = useState({})
+
+const [username, setUsername] = useState('')
+const [email, setEmail] = useState('')
+
+
+const CREATE_USER = gql`
```

Comment:
Glad we were able to get this mutation working correctly!

Reply:
yes, it took a while to resolve, but now we know the syntax required.